### ISSUE TITLE
Generating answers from a Q&A training set

### ISSUE THREAD

**Issue by jrthom18** [jcjohnson/torch-rnn#153]
Say I train the RNN with a data set of questions and answers on a closed-knowledge domain. How can I test the model to see if it can successfully answer the questions that it saw in training? Obviously sampling generates random output that gives you the flavor of the training data so I can't just seed the sampling script with a test question and expect it to give the correct answer back.
